beatrix/src/test/java/org/killbill/billing/beatrix/integration/TestCatalogFixedTerm.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2020-2020 Equinix, Inc
+ * Copyright 2014-2020 The Billing Project, LLC
+ *
+ * The Billing Project licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.killbill.billing.beatrix.integration;
+
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertNull;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import org.joda.time.DateTime;
+import org.joda.time.LocalDate;
+import org.killbill.billing.account.api.Account;
+import org.killbill.billing.account.api.AccountData;
+import org.killbill.billing.api.TestApiListener.NextEvent;
+import org.killbill.billing.beatrix.util.InvoiceChecker.ExpectedInvoiceItemCheck;
+import org.killbill.billing.catalog.api.CatalogApiException;
+import org.killbill.billing.catalog.api.PlanPhaseSpecifier;
+import org.killbill.billing.entitlement.api.DefaultEntitlementSpecifier;
+import org.killbill.billing.invoice.api.Invoice;
+import org.killbill.billing.invoice.api.InvoiceItemType;
+import org.killbill.billing.payment.api.PluginProperty;
+import org.killbill.billing.util.callcontext.CallContext;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import com.google.common.base.Charsets;
+import com.google.common.collect.ImmutableList;
+import com.google.common.io.Resources;
+
+public class TestCatalogFixedTerm extends TestIntegrationBase {
+
+    private CallContext testCallContext;
+    private Account account;
+
+    @BeforeMethod(groups = "slow")
+    public void beforeMethod() throws Exception {
+        if (hasFailed()) {
+            return;
+        }
+
+        super.beforeMethod();
+
+        // Setup tenant
+        clock.setTime(new DateTime("2021-12-01T12:56:02"));
+        testCallContext = setupTenant();
+
+        // Setup account in right tenant
+        account = setupAccount(testCallContext);
+    }
+
+    @Test(groups = "slow", description = "https://github.com/killbill/killbill/issues/1533")
+    public void testFixedTermPhaseAndRecurringBillingPeriod() throws Exception {
+        uploadCatalog("fixedterm-and-biennial-billing.xml");
```

Comment:
I would suggest to use a different name for your catalog file, so that if/when we add more `Plans` (in addition to the existing `p1-biennial`), the name still makes sense.

Reply:
@sbrossie, would you prefer if additional `Plans` are added to the same catalog, or would you prefer creating different catalog files for each test as done for other tests within the [/src/test/resources/catalog](https://github.com/killbill/killbill/tree/master/beatrix/src/test/resources/catalogs) folder? If you would prefer adding multiple plans to the same file, I will go ahead and rename the catalog file.

Reply:
I think it depends on the use cases - I would say, if you don't need multiple versions, stay out of it, and have multiple Product/Plans in one catalog version.

Reply:
Got it, I've now renamed the catalog and added more plans to it.